### DISCUSSION TITLE
How to align the footer to the center?

### DISCUSSION THREAD

**Discussion by gautamigolani** [alshedivat/al-folio#257] [Q&A] [answered]
Thanks for the great theme!

I am trying to align the footer in my webpage to the center but have not succeeded in doing so as in the picture below. 

<img width="914" alt="footer_center" src="https://user-images.githubusercontent.com/47815411/116740291-659a7580-aa27-11eb-8091-152ea7ab7943.png">

I have tried inserting 'text-align: center' under footer.sticky-bottom in _base.scss but it does not work and I am not sure if this is the correct method. Could you please advise on this.

Thank you

**Comment by alshedivat** [comment, marked as answer]
Good question! Try adding `text-align: center` to the div inside `footer.sticky-bottom`, like this:
```css
footer.sticky-bottom div {
  text-align: center;
}
```

**Reply by gautamigolani** [reply]
This works perfectly fine! 
Thanks for helping out.

**Reply by masoudr** [reply]
I tried to change this in `_base.scss` but didn't work.
```
footer.sticky-bottom {
  border-top: 1px solid var(--global-divider-color);
  padding-top: 40px;
  padding-bottom: 40px;
  font-size: 0.9rem;
  text-align: center;
}
```

**Reply by rajanguluri** [reply]
`text-align: center;` worked for me when I added It in `footer.fixed-bottom {}` environment.
